packages/core/src/WebSocket.ts

```diff
@@ -13,32 +12,3 @@ export interface WSMessage {
   /** Latest value(s) for the `NetworkInfoProvider` methods.*/
   networkInfo?: Partial<NetworkInfoResponses>;
 }
-
```

Comment:
Maybe we should also hoist `WsMessage` as it's intended to be used with `WsProvider`
Also, please reword the commit to indicate breaking change

Reply:
72e42cebb0a should solve this